tests/RedisConnectionTest.php

```diff
@@ -87,6 +93,29 @@ public function testSerialize()
         $this->assertTrue($db2->ping());
     }
 
+    public function testConnectionTimeout()
+    {
+        $db = $this->getConnection(false);
+        $db->configSet('timeout', 1);
+        $this->assertTrue($db->ping());
+        sleep(1);
```

Comment:
I wonder if these tests pass, since you have a timeout of 1 (sec?) and you sleep 1 seconds, plus a few microseconds for code-execution... Wouldn't it be better to lower this to 0.5?

Reply:
We could change it if needed but I have not seen any issues with this, neither locally nor on travis. Will improve tests a bit.